.github/workflows/test.yml

```diff
@@ -22,7 +22,16 @@ jobs:
           source activate black
           pip install black
           black --check urlchecker
-
+          
+          
+  type_checking:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpetrucciani/mypy-check@0.930
```

Comment:
For others actions I usually pin an exact commit, put a comment about the version it is, and then do a quick security review of it.

Reply:
Also what this action does is SUPER simple, can we not just do that instead?

https://github.com/jpetrucciani/mypy-check/blob/ccc60aacd735c6bf77cc3ddcc8df963282042acd/entrypoint.sh#L1-L18

Reply:
This was lazy on my side. I am fixing this right away ;)